src/main/java/ivanov/springbootintro/service/BookService.java

```diff
@@ -7,15 +7,15 @@
 import org.springframework.data.domain.Pageable;
 
 public interface BookService {

```

Comment:
previous naming was more common TBH, it's bookService, we don't need to mention "book" in every method